### PR TITLE
Rename py-sgit to subgit on PyPI

### PR DESCRIPTION
**Pull request by holmboe**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
     entry_points={
         "console_scripts": [
             "sgit = sgit.cli:cli_entrypoint",
+            "subgit = sgit.cli:cli_entrypoint",
```
Comment (Grokzen) [inline setup.py:22]: @holmboe Please remove the script `git-sub` with this new added we do not need to have three diffferent ones for now.

Comment (holmboe) [reply setup.py:22]: @Grokzen It is rather common to extend git with subcommands in this manner, see https://medium.com/@santiaro90/write-your-own-git-subcommands-36d08f6a673e or https://blog.sebastian-daschner.com/entries/custom-git-subcommands

Do you still think it is an unnecessary convenience?

If you ask me I think we should make `git sub` the main entry point (in documentation) and keep `sgit` and/or `subgit` as convenience aliases.

Comment (Grokzen) [reply setup.py:22]: If we are keeping the git command implementation i still think we should stick with two and not three in our setup.py. Three is to many to have just becuase

Comment (holmboe) [reply setup.py:22]: Converting to draft to pick up on this after we have merged some conflicting PR:s.

**Comment by Grokzen** [comment]
PR superseeded by #30

**Pull request closed without merging**